rust/private/rust.bzl

```diff
@@ -90,6 +90,8 @@ def _determine_lib_name(name, crate_type, toolchain, lib_hash = ""):
     prefix = "lib"
     if (toolchain.target_triple.find("windows") != -1) and crate_type not in ("lib", "rlib"):
         prefix = ""
+    if toolchain.target_arch == "wasm32" and crate_type == "cdylib":
+        prefix = ""
```

Comment:
Not your problem, and this doesn't make it any worse, but the logic for "prefix" needs to be cleaned up in this function.

Reply:
Yeah, I wanted to avoid refactoring while fixing things so did it in the existing style.